profiles/SampleBarbarian-setup.yaml

```diff
@@ -1,131 +1,604 @@
+
+buff_nonspells:
+  roar quiet wail: 90 #roars on timer
+  barb_buffs: #maintains buffs
+#meds
+  #- Serenity
+  #- Contemplation
+  - Tenacity
+#forms
+  #- Turtle
+  #- Panther
+  #- Python
+  - Piranha
+  - Eagle
+  - Dragon
+  - Monkey
+  #- Swan
+#bers
+  #- Famine
+  - Tornado
+  #- Earthquake
+  - Wildfire
+  - Flashflood
+  #- Landslide
+
+#######################################################################################################
+
+#waggles and anchors are advanced options - only use if you are comfortable
+#these are the basic spells, add others as needed
+waggle_sets:
+  #will be used for prehunt, standard, and combat buffs
+  default: &buffs
+  #- Python
+  - Piranha
+  - Eagle
+  - Dragon
+  - Monkey
+  - Tornado
+  - Wildfire
+
+####################################################################################################
+
+#turn turn this on to avoid boxes
+# custom_loot_type: treasure
+# box_loot_limit: 0
+
+#non-standard loot to pick up - anything outside of gems/boxes/maps need to be added
+loot_additions:
+#- item1
+#- item2
 
-# Gear settings
+#standard loot to not pick up - typically for certain types of ammo you don't want to pick up from buddies
+loot_subtractions:  
+#- item1
+#- item2
+
+#store specific items in specific containers
+loot_specials:
+# - name: jadeite stones
+#   bag: satchel
+# - name: cigarillo
+#   bag: satchel
+# - name: scroll
+#   bag: haversack
+# - name: parchment
+#   bag: haversack
+
+#where to store full loot pouches
+full_pouch_container: lootsack
+#separate location for spare empty pouches
+spare_gem_pouch_container: backpack
+
+#true to save pouches for trader to sell for better profit
+# false to sell yourself
+tie_gem_pouches: true
+
+#pick a specific gem pouch adjective.  
+#DO NOT USE "GEM"! It will eventually break and you'll end up in the junk yard
+gem_pouch_adjective: dark
+
+# Sell loot settings
+sell_loot_pouch: false
+sell_loot_bundle: true
+sell_loot_money_on_hand: 3 silver
+
+#hunting-buddy will maintain a minimum of storebought ammo - will need more details outside crossing
+restock:
+  arrow:
+    quantity: 15
+  bolt:
+    quantity: 15
+  rock:
+    quantity: 15
+  #You can specify custom restocking, advanced feature
+  # rock:
+  #   hometown: Hibarnhvidar
+  #   name: smooth rocks
+  #   room: 12172
+  #   price: 45
+  #   size: 30
+  #   stackable: true
+  #   quantity: 30
+
+# Lockpicking settings - follow it carefully to avoid springing traps!
+stop_pick_on_mindlock: true  
+pick_blind: false
+use_lockpick_ring: true
+skip_lockpick_ring_refill: false
+lockpick_container: lockpick ring
+lockpick_type: stout  #use 'stout iron' if in Shard
+harvest_traps: false 
+#live box source
+picking_box_source: pack
+# Keep this empty to drop too-hard boxes on the ground
+picking_box_storage:
+#pet box source
+picking_pet_box_source: satchel
+#number of boxes to make with ;pick pets
+pet_boxes_on_hand: 10
+
+#settings for lockpick training box to use with ;lockbox
```

Comment:
Maybe update this to the new ;locksmithing settings?  And add burgle?  Since neither was available when the sample empath file came out.

Reply:
What are the new locksmithing settings? 

I will add burgle